### DISCUSSION TITLE
Saint Lawrence, 10 of august

### DISCUSSION THREAD

**Discussion by Padre-Anselmo** [DivinumOfficium/divinum-officium#4163] [Developers corner]
When saint Lawrence is on a Sunday, then there is only commemoration in Laudes
AND (!!!)
there will NOT be a Vigil of this saint - IMHO (that's what I had learned a long time ago).
How should this be corrected in the code? Could one of you do that.
This situation will take place next year.
Thank you,

fr Anselm

**Comment by APMarcello3** [comment]
4e17cb301abe4f80f2b1294e40f40ca351120df4

**Reply by FAJ-Munich** [reply]
Isn't the vigil only scrapped if the vigil falls on Sunday? That would be Dominical letter f and 2026 where it is already correct. What needs to be removed is the notorious "2nd vespers of the Vigil" In concurrence with the first vespers of the Sunday.

**Reply by FAJ-Munich** [reply]
Sorry, I retract what I said

> 33. A vigil of the 2nd or 3rd class is omitted altogether if it occurs on any Sunday or on a feast of the 1st class, or if the feast to which it is prefixed happens to be transferred to another day or reduced to a commemoration.
